vnext/Desktop.IntegrationTests/Modules/TestDevSettingsModule.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "TestDevSettingsModule.h"
+
+using namespace facebook::xplat;
+using namespace folly;
+
+using std::function;
+using std::map;
+using std::string;
+using std::vector;
+
+namespace Microsoft::React::Test {
+
+/*static*/ string TestDevSettingsModule::name = "DevSettings";
+
+#pragma region CxxModule methods
+
+string TestDevSettingsModule::getName() {
+  return name;
+}
+
+map<string, dynamic> TestDevSettingsModule::getConstants() {
+  return {};
+}
+
+vector<module::CxxModule::Method> TestDevSettingsModule::getMethods() {
+  return {
+      {"addMenuItem",
+       [this](dynamic args) {
```

Comment:
>args [](start = 22, length = 4)

Please avoid declaring unused variables. I guess somewhere we have the compiler warning suppression which we must remove for our code.

Reply:
Note: If it is for lifetime purposes, please add comment to indicate so.

Reply:
A common pattern here is to just comment out the param name. E,g,g `(dynamic /*args*/)`